vendor/JSON-Schema-Test-Suite/.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Test Suite Sanity Checking
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [published]
+  schedule:
+    # Daily at 6:42, arbitrarily as a time that's possibly non-busy
+    - cron: '42 6 * * *'
```

Comment:
Is there a reason for a recurrent schedule, additionally to checking on push & pull_request?

Reply:
This is part of the vendored source form `JSON-Schema-Test-Suite`, it shouldn't affect our CI at all.